hypha/apply/activity/templates/messages/email/vendor_setup_needed.html

```diff
@@ -0,0 +1,11 @@
+{% extends "messages/email/applicant_base.html" %}
+
+{% block content %}
+A Project has been created for your submission on {{ ORG_SHORT_NAME }}.
+
+Next step is to complete vendor setup form by visiting project detail page.
```

Comment:
@frjo, was not sure about the correct statement in the email template. Let me know your suggestions.

Note that we send [another email](https://github.com/HyphaApp/hypha/blob/main/hypha/apply/activity/templates/messages/email/determination.html) when we do determination. So on approve both emails will be sent.

Reply:
@ei8fdb Will work on the wording in the mail. A link directly to the vendor form should be included as well.